cmd/geth/usage.go

```diff
@@ -237,18 +229,18 @@ var AppHelpFlagGroups = []flags.FlagGroup{
 			utils.CatalystFlag,
 		},
 	},
-}
+},
```

Comment:
revert once the upper bracket is reinstated

Reply:
Fixed!